Makefile

```diff
@@ -9,10 +10,10 @@ luacheck:
 .PHONY: test
 test:
 	.rocks/bin/luatest -v
-	rm -rf *.xlog* *.snap 51{2,3,4,5,6,7}
+	rm -rf ${CLEANUP_FILES}
 	INDEX_TYPE='TREE' SPACE_TYPE='vinyl' ./test.lua
-	rm -rf *.xlog* *.snap 51{2,3,4,5,6,7}
+	rm -rf ${CLEANUP_FILES}
 	INDEX_TYPE='HASH' ./test.lua
-	rm -rf *.xlog* *.snap 51{2,3,4,5,6,7}
```

Comment:
`51{2,3,4,5,6,7}` is about directories that vinyl creates. Why aren't they in CLEANUP_FILES?

Reply:
I thought it is a set of files for sophia or other legacy.